src/app.rs

```diff
@@ -415,6 +415,14 @@ pub fn build_app() -> App<'static, 'static> {
                 .hidden_short_help(true)
                 .long_help("Limit the number of search results to 'count' and quit immediately."),
         )
+        .arg(
+            Arg::with_name("max-one-result")
+                .short("1")
+                .hidden_short_help(true)
+                .overrides_with("max-results")
+                .conflicts_with_all(&["exec", "exec-batch"])
+                .long_help("Short form of '--max-results=1'.")
```

Comment:
Even if it is just the short form for `--max-results=1`, I think it would be good to repeat what it does. Also, we use the word "alias" throughout the `--help` text. So maybe:
```rust
.long_help("Limit the search to a single result. This is an alias for '--max-results=1'.")
```
What do you think?

(same for the man page entry)

Reply:
I agree. Good point.